### PR TITLE
#44 Added error pages

### PR DESCRIPTION
**Pull request by DanGravel**

### CODE REVIEW

**Review by Taz-Z — APPROVED** [review]

File: Project/src/main/resources/static/css/error.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    }
+    75% {
+        color: #F56991;
+    }
+    100% {
+        color: #FFC48C;
+    }
+}
+
+/* demo stuff */
```
Comment (liamhickey) [inline Project/src/main/resources/static/css/error.css]: What's this?

File: Project/src/main/resources/templates/error/404.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>404</title>
+    <link rel="stylesheet" th:href="@{/css/error.css}" />
+</head>
+<body>
+    <section class="error-container">
+        <span>404</span>
+    </section>
```
Comment (liamhickey) [inline Project/src/main/resources/templates/error/404.html:11]: Could add a short description here

Comment (DanGravel) [reply Project/src/main/resources/templates/error/404.html:11]: for?

**Review by liamhickey — APPROVED** [review]
+1

**Pull request merged**